samples/contrib/aws-samples/simple_train_pipeline/training-pipeline.py

```diff
@@ -53,7 +53,7 @@ def training(
         instance_count=1,
         volume_size=50,
         max_run_time=3600,
-        model_artifact_path='s3://kubeflow-pipeline-data/mnist_kmeans_example/data',
+        model_artifact_path='s3://kubeflow-pipeline-data/mnist_kmeans_example/output',
```

Comment:
should a test have caught this ?

Reply:
As discussed, this isn't a bug but a cleanup.

Reply:
ya it's a cleanup 
and integ test wouldn’t have caught it even if it was a bug coz they give their own input and override this default